### PR TITLE
✨  Bento LightboxGallery Navigational UI

### PR DESCRIPTION
**Pull request by caroqliu**
This PR adds the following user interface features:
- Display lightboxed elements in a carousel in the open gallery
- Custom close button identical to the 0.1 SVG
- Custom carousel arrows identical to the 0.1 SVGs
- Unit tests

A follow up PR will add the togglable grid view. Partial for #34737

### CODE REVIEW

File: extensions/amp-lightbox-gallery/1.0/consumer.js [hunk 1]
```diff
@@ -56,14 +55,10 @@ export function WithLightbox({
     () =>
       enableActivation && {
         ...DEFAULT_ACTIVATION_PROPS,
-        onClick: () => {
-          if (customOnClick) {
-            customOnClick();
-          }
-          open();
-        },
+        /* genKey is 1-indexed, gallery is 0-indexed */
```
Comment (dmanek) [inline extensions/amp-lightbox-gallery/1.0/consumer.js:58]: 🤦🏽  thanks for comment explaining this

**Review by dmanek — APPROVED** [review]

File: extensions/amp-lightbox-gallery/1.0/component.jss.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+import {createUseStyles} from 'react-jss';
+
+const gallery = {
+  position: 'absolute !important',
+  left: '0 important',
+  right: '0 important',
+  top: '0 !important' /* Matches height of top-bar */,
+  height: '100%',
+  width: '100%',
+  bottom: '0 important',
```
[the remark this answers is not in the document]
Comment (caroqliu) [reply extensions/amp-lightbox-gallery/1.0/component.jss.js:26]: Thanks

**Review by alanorozco — APPROVED** [review]

File: extensions/amp-lightbox-gallery/1.0/component.jss.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+const topControl = {
+  width: '24px',
+  height: '24px',
+  padding: '16px',
+  '@media (min-width:1024px)': {
+    width: '40px',
+    height: '40px',
+    padding: '20px',
+  },
```
Comment (alanorozco) [inline extensions/amp-lightbox-gallery/1.0/component.jss.js:72]: nit: these are all pixels by default when numbers
```suggestion
  width: 24,
  height: 24,
  padding: 16,
  '@media (min-width:1024px)': {
    width: 40,
    height: 40,
    padding: 20,
  },
```

Consider also sharing a constant for `20` or `40` across properties.

Comment (caroqliu) [reply extensions/amp-lightbox-gallery/1.0/component.jss.js:72]: Thanks, done!

**Pull request merged**